tests/Behat/features/compare-mode.feature

```diff
@@ -10,8 +10,11 @@ Feature: Compare mode
 
     Given I am logged in with "ADMIN" permissions
     And I go to "/admin/pages"
+    And I wait 3 seconds
```

Comment:
Ugh this will eat up our (now more precious) build minutes, is it really that flakey? Why wasn't it before this PR?

Reply:
I don't know. It was really frustrating and a huge expenditure of time. Happens with status quo and with graphql 4. Might be something we can try removing later on and see what happens.

Reply:
Fair enough, not worth our time right now, and I see you've put the "To Behat or not to Behat" question back on the agenda.